### ISSUE TITLE
AVT 3 - Tab Component: State and correct role are not announced by VoiceOver

### ISSUE THREAD

**Issue by snidersd** [IBM/carbon-components-react#2222]
## Environment
macOS Mojave version 10.14.2
Chrome version 72.0.3626.96 (Official Build) (64-bit)

## Detailed Description
VO announces does not announce the correct role for each button and does not announce when it is selected. 

## Expected Result
VO should announce each button as a tab using role="tab" and indicate when it is selected using aria-selected="true".
The Right and Left Arrow keys should move focus forward and backward through the tabs.

See [ARIA Authoring Practices 1.1 - Tabs ](https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-1/tabs.html) for additional ARIA requirements.
It is also recommended that the example include the tabpanel to ensure the ARIA roles and states, as well as focus are implemented correctly.

**Comment by snidersd** [comment]
This issue should remain open until it is fixed and retested.
